routes.js

```diff
@@ -77,6 +77,9 @@ module.exports = (app) => {
   // TODO(tony): change it back to /restaurants/me
   router.put("/restaurants/:id", controller.updateRestaurant);
   router.post("/restaurants/register", controller.createRestaurant);
+  router.post("/restaurants/create-group", controller.createRestaurantGroup);
+  router.post("/user/add-to-group", controller.addUserToGroup);
```

Comment:
What's the use case of `/user/add-to-group`? @cchyung 

If it's for adding the group creator to the group, maybe we'd like to combine this with `/restaurants/create-group` for atomicity. If it's for managing group members, something like`/user/:userId/add-to-group/:groupId` would be better.

Reply:
I think it makes more sense to pass userID as a parameter because the scenario would be a different user adding a given user to a given group.  

@clairepo can we change it to `/groups/:groupID/add-to-group` and then take the userID in the body?

Reply:
Create group is done at registration now, when a user first signs up for the nomi service

Reply:
In the create group endpoint, should the user creating the group automatically be added to the group?

Reply:
Yeah I think that would make sense!

Reply:
Ok now Im thinking, should the create group endpoint also take a restaurantId and add a restaurant to it?

Reply:
I also made the restaurantGroupId allowNull to be true for user and restaurant, but I'm not 100% sure about that. If creating a group is done when a user signs up, then they will always be part of a group right?

Reply:
I think we should be good because the group gets created at registration time, but the restaurant itself isn't created until after the group gets created according to our registration flow, so we will probably want to make a method that's llike "`restaurant/add-to-group` that let's us add a specific restaurant to a group after it's been created, does that make any sense lol